linux_os/guide/system/software/integrity/fips/enable_fips_mode/rule.yml

```diff
@@ -26,8 +26,6 @@ rationale: |-
 
 severity: high
 
-platform: machine  # The check uses syscyl_... extended definition, which doesnt support offline mode
```

Comment:
Limitation of OpenSCAP. Content needs to be tool agnostic. Reverting.

Reply:
ditto (see `sysctl_kernel_exec_shield`)